src/components/VLAutocomplete/types.ts

```diff
@@ -1,3 +1,10 @@
+import type {
+  AutoCompleteChangeEvent,
+  AutoCompleteCompleteEvent,
+  AutoCompleteDropdownClickEvent,
+  AutoCompleteItemSelectEvent,
+  AutoCompleteItemUnselectEvent
+} from 'primevue/autocomplete'
```

Comment:
Better to import them from utils/types cause the user can need the events type from us and not the peer dependence